### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by nmarcetic** [mainflux/devops#24]
Improve getting started documentation, step by step guide how to install charts on running clusters.
Its simple and easy flow, let's document it well. 

FYI There is no consistent way to install Nginx ingress controller, some providers have it already somewhere you need to install it manually, let's document this for AWS, GCE, and Digitalocean at least.

**Comment by drasko** [comment]
@blokovi I think this one can be closed, right?

**Comment by blokovi** [comment]
I would leave this open until ingress documentation is ready. I think it's good idea to have this documented for diffrent providers

**Comment by nmarcetic** [comment]
I agree with @blokovi at least for 3-4 top cloud providers with managed k8s.

**Comment by CD-UNCC** [comment]
agreed. could really use help getting this to work on an existing k8s cluster with an existing nginx ingress controller on Azure.
